configs/components/_base-ruby.rb

```diff
@@ -140,11 +116,27 @@
   #                   -Sean P. McDonald 07/01/2016
   lib_type = platform.architecture == "x64" ? "seh" : "sjlj"
```

Comment:
Looks like you replaced usage of `lib_type` with `gcc_postfix` - should this line be removed?